backend/app/controllers/answers_surveys_controller.rb

```diff
@@ -1,10 +1,19 @@
 class AnswersSurveysController < ApplicationController
   def create
```

Comment:
what about this?
```
answers_survey = AnswersSurvey.create!(create_answers_survey_params)
redirect_to answers_survey.survey.questions.first
```

Reply:
https://github.com/espoo-dev/espoo-dev/pull/403/commits/89179ad0a306f1283c0d938383c8b058ebb3625e